### ISSUE TITLE
[Bug][A11Y] Checkbox: hint is missing on component

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#391]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone

### OS Version

iOS

### App version

0.6.0

### Describe the issue

The checkbox component is read like this "checkbox, not checked". The hint is missing to help users understanding how to interact

### Expected Behavior

add a hint "double tap to toggle setting checked" (to be consistent with iOS app)

**Comment by nouha06** [comment]
@pylapp you can see my comment here https://github.com/Orange-OpenSource/ouds-flutter/issues/393#issuecomment-3416142627

**Comment by pya35** [comment]
@nouha06 see Android Compose techniques for inspiration: https://github.com/cvs-health/android-compose-accessibility-techniques/blob/main/doc/components/CheckboxControls.md

**Comment by nouha06** [comment]
The issue has been fixed and the ticket has been closed, as we cannot release a new version. That’s why the TestFlight blocking label was applied.

**Comment by B3nz01d** [comment]
Closed during Sprint # 23

**Comment by nouha06** [comment]
📣 New Firebase App Distribution BETA upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: DesignToolbox (BETA) (485)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 442
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
New TestFlight BETA upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@Tayebsed93) to get access.

Display name: Design Toolbox Flutter (485)
Version: 0.6.0
Build type: BETA
Build details (GitHub): 442
cc @B3nz01d (product owner)
cc @Tayebsed93 @AhmedAmineZr @nouha06 (dev team)
cc @jerome-regnier (design team)
cc @pya35 (accessibility team)

**Comment by pya35** [comment]
@nouha06 reopened because:

- [x] - on Android: hint is "double-tap to activate" - it should be "double-tap to toggle" 

- [x] - on both apps: hint is missing for checked state -  it should be "double-tap to toggle" also

**Comment by B3nz01d** [comment]
@nouha06 where can this be tested?

**Comment by nouha06** [comment]
> [@nouha06](https://github.com/nouha06) where can this be tested?

in build version 589 it's included in the parent ticket #514

**Comment by pya35** [comment]
will be checked once #514 will be fixed
